services/web-app/pages/assets/[host]/[org]/[repo]/[library]/[ref]/[asset].js

```diff
@@ -88,44 +133,102 @@ const Asset = ({ libraryData }) => {
             aspectRatio={{ sm: '2x1', md: '1x1', lg: '3x4', xlg: '1x1' }}
             border={['sm']}
           >
-            Maintainer
+            <Column className={dashboardStyles.title}>Maintainer</Column>
```

Comment:
Why columns here? Does anything need laid out horizontally next to each other. I'm thinking semantic typography elements can work here like `<p>` and a heading `<h3>`.